common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -1197,15 +1230,15 @@ public void onData(String streamId, ByteBuffer buf) throws IOException {
             appShuffleInfo.shuffles.get(partitionInfo.appAttemptShuffleMergeId.shuffleId);
         if (isStale(info, partitionInfo.appAttemptShuffleMergeId.shuffleMergeId) ||
             isTooLate(info, partitionInfo.reduceId)) {
```

Comment:
If too late, mark `lateBlockPushes` ?

Reply:
Any thoughts ?

Reply:
Yes, we should mark `lateBlockPushes` here, thanks for the suggestion!